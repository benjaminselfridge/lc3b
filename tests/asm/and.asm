        0x3010
        ADD     R0      R0      0x0A
        ADD     R1      R1      0x05
        AND     R2      R1      R0
        ADD     R1      R1      0x02
        AND     R3      R1      R0
        TRAP    0x0
