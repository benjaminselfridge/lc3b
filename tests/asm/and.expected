[ RegContains 0 0x0A
, RegContains 1 0x07
, RegContains 2 0x00
, RegContains 3 0x02
]